var/docs/docker.md

```diff
@@ -1,5 +1,7 @@
 # Docker
 
+There is an official docker repo for the kimai app, https://github.com/tobybatch/kimai2 and a dockerhub repo https://hub.docker.com/r/kimai/kimai2.  Please raise any container issues [there](https://github.com/tobybatch/kimai2/issues).
```

Comment:
Probably rephrase to something like `There is an official docker repo for Kimai 2 production usage`, just to make sure that the difference between these two docker images is clear for everyone who wasn't involved.

Reply:
Wait, its not only for production usage, but for dev also? Then it would probably be even more confusing what I wrote...

But could you please make proper links out of the URLs, so they work also when rendered directly in Kimai on the "Manual" page.

Reply:
I've re-worded it and I think fixed the links as you want.

Reply:
Cool, thanks Toby!